### DISCUSSION TITLE
Code block copy button

### DISCUSSION THREAD

**Discussion by kaysauter** [thuliteio/doks#350] [Q&A] [answered]
Hi, 

the code blocks as in the doks documentation site [here](https://getdoks.org/docs/extensions/highlight-js/) don't seem to come with a copy-button like in the linked site. How do I activate this copy-button? It seems to me that this is not documented.

**Comment by h-enk** [comment, marked as answer]
Hi @kaysauter, the new copy button is available in the starter theme already, in a few hours also in the child theme. No config required. Will add switch later, if wanted. 

I will keep you updated.

**Reply by h-enk** [reply]
@kaysauter, I've just released Doks [v0.3.0](https://github.com/h-enk/doks/releases/tag/v0.3.0) (including the new copy button a/o) + @hyas/doks [v0.3.0](https://www.npmjs.com/package/@hyas/doks/v/0.3.0) + Doks child theme [v0.3.0](https://github.com/h-enk/doks-child-theme/releases/tag/v0.3.0).
